### PR TITLE
Fixed iOS Voice issue

### PR DESCRIPTION
**Pull request by martinglass**
Added lines to automatically trigger voice for iOS on "Run" page, when Start button or Schedule Race button is pressed.
Also added code for "Current" page, to provide a button to Initialize iOS voice, at the bottom of the page

### CODE REVIEW

**Comment by HazardCreative** [comment]
I have some additional testing and less invasive methods I want to try before committing to a final solution.

**Comment by martinglass** [comment]
> I have some additional testing and less invasive methods I want to try before committing to a final solution.

Sure - I've just put it up in case it's useful to people right now.

File: src/rh/templates/current.html [hunk 1]
```diff
@@ -797,6 +802,6 @@ <h2>{{ __('Display Settings') }}</h2>
 		</ol>
 	</div>
 </div>
-
+<button class="button" id="voice_initialize">Initialize Voice (iOS)</button>
```
Comment (pulquero) [inline src/rh/templates/current.html:805]:
```suggestion
<button class="button" id="voice_initialize">Initialize Voice</button>
```

**Comment by martinglass** [comment]
@HazardCreative has created a cleaner, more robust solution. See ios-audio branch in RH
https://github.com/RotorHazard/RotorHazard/tree/ios-audio

**Pull request closed without merging**